src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d", // event id
+  "type": "GradeEvent",
+  "actor":{
+    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171", // student id
+    "type":"Person"
+  },
+  "action": "Graded",
+  "object": {
+    "id": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f", // last attempt id
+    "type": "Attempt",
+    "assignee": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171", // student id
+    "assignable": {
+      "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+      "type": "Assessment"
+    },
+    "count": 35,
+    "dateCreated": "2016-11-15T10:05:00.000Z",
+    "startedAtTime": "2016-11-15T10:05:00.000Z",
+    "endedAtTime": "2016-11-15T10:55:12.000Z"
+  },
+  "eventTime": "2016-11-15T10:57:06.000Z",
+  "edApp": "https://www.hmhco.com/PartnerService",
+  "generated": {
+    "id": "urn:uuid:2050e852-5edb-4743-92d1-b53466de3a5f", // random UUID
+    "type": "Score",
+    "attempt": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f", //last attempt id
+    "maxScore": 1400,
+    "scoreGiven": 500,
+    "scoredBy": "https://www.hmhco.com/PartnerService",
+    "comment": "The Assessment score from RenLearn # c35603f7a3434e2cb74ef14d60e70f42 taken on date/time for student 0f4fedbe-2227-415f-8553-40731a627171",
+    "dateCreated": "2016-11-15T10:56:00.000Z"
+  },
+  "membership":{
+    "id":"urn:uuid:8f4fedbe-2227-415f-8553-40731a627171", // random UUID
+    "type":"Membership",
+    "organization":{
+      "id":"urn:uuid:ab570d90-3791-4048-a5ee-759657ddccef", // random UUID
+      "type":"Organization",
+      "subOrganizationOf":{
+        "id":"urn:uuid:ea827de8-0ce9-4fe4-a28c-eb388bd9eb92", // random UUID
```

Comment:
this isn't a random id, it must be the district ref id that is associated with the school ref id above it

Reply:
Sure. I can add the district ref id for the school which student is associated in.